addons/l10n_cl/models/res_partner.py

```diff
@@ -21,3 +22,34 @@ class ResPartner(models.Model):
              '2 - Fees Receipt Issuer (Applies to suppliers who issue fees receipt)\n'
              '3 - End consumer (only receipts)\n'
              '4 - Foreigner')
+
+    def _format_vat_cl(self, values):
```

Comment:
No mentioning of this in the commit message.

Reply:
perhaps because it pertains to a very old part and not the last modifications. Editing the pr/commit message